specification.markdown

```diff
@@ -60,6 +60,8 @@ This document defines specific terms in order to define the format and applicati
 
 `Descendant Element`: A `Descendant Element` is a relative term to describe any `EBML Elements` contained within a `Master Element`, including any of the `Child Elements` of its `Child Elements`, and so on.
 
+`Void Element`: A `Void Element` is an element used to void damaged data or reserve space in a sub-element for later use.
```

Comment:
I looked through the rest of the spec to see what I could come up with on my own, just saw that it can be but is not considered officially to be a `Root Element`, so seems like it isn't necessarily a `Child` either?